diff_pylint.py

```diff
@@ -0,0 +1,11 @@
+import subprocess
```

Comment:
I would move this file into the .workflows directory

Reply:
done.